includes/subscriptions/class-wc-payments-product-service.php

```diff
@@ -113,7 +113,7 @@ public static function get_wcpay_price_hash( WC_Product $product ) : string {
 	 * @param WC_Product $product The product to get the WC Pay price ID for.
 	 * @return string             The product's WC Pay price ID or an empty string.
 	 */
-	public static function get_wcpay_price_id( WC_Product $product ) : string {
+	public function get_wcpay_price_id( WC_Product $product ) : string {
```

Comment:
Just noting that in order for us to mock this method in unit tests, I had to make this non-`static`. This is in large part due to the limitations of subscriptions/subscription products in our unit tests. There might be a better way to handle this, so let me know if you can think of one.